README.md

```diff
@@ -76,6 +76,16 @@ To manually send a span:
 The following auto-instrumentation libraries are automatically included:
 * [MetricKit](https://developer.apple.com/documentation/metrickit) data is automatically collected.
 
+### UIKit instrumentation
```

Comment:
nit: It looks like we capitalized "Instrumentation" everywhere else, so let's be consistent.